pkg/trace/stats/aggregation.go

```diff
@@ -10,6 +10,8 @@ const (
 	tagHostname   = "_dd.hostname"
 	tagStatusCode = "http.status_code"
 	tagVersion    = "version"
+	tagOrigin     = "_dd.origin"
+	tagSynthetic  = "synthetic"
```

Comment:
Why is the tag called syntheticrather than synthetics like in _dd.origin ?

Reply:
That's my mistake. I'll fix it. Thank you!